src/json.rs

```diff
@@ -459,7 +459,9 @@ impl JsonEmitter {
                 write!(
                     w,
                     "{}",
-                    self.color.comment.paint(format!("{} {}", leader, line))
+                    self.color
+                        .comment
+                        .paint(format_args!("{} {}", leader, line))
```

Comment:
is `format_args` quicker when the result is going to be an argument of another formatter?

Reply:
Yes, it allows formatting into final destination directly without going through another `String`.

Reply:
It's super cool. I was unaware of this.